projects/documentation/src/components/search-index.ts

```diff
@@ -56,10 +56,14 @@ export async function search(value: string): Promise<ResultGroup[]> {
     const search = index.search(value);
     for (const item of search) {
         const parts = item.ref.split('/');
-        if (parts.length < 3) continue;
-
         const name = parts.pop() as string;
-        const category = parts.pop() as string;
+        let category = '';
+        if (parts.length < 3) {
+            category = 'Guides';
+        } else {
+            category = parts.pop() as string;
+        }
+
```

Comment:
Wondering if it would be possible to add the category as metadata in `build-search-index.js` rather than deriving it from the ref in this manner which seems a bit fragile to me

Reply:
Good idea! `lunr.js` is a little wonky with the data it allows you to get back form the search query, but I've added a stringified JSON into the item refs so that we can get all of this data back without too much trouble. I also see that `lunr.js` hasn't been updated in quite a while...somethings "just work", but it's a decent sign that at _some point_ we might want to look into alternative solutions for our search processing needs.